components/runtimes/Makefile

```diff
@@ -3,33 +3,11 @@ ROOT :=  $(shell pwd)
 release: build-images push-images
 
 .PHONY: build-images
-	build-function-runtime-nodejs16-image \
 	build-function-runtime-python39-image
 
 .PHONY: push-images
-	push-function-runtime-nodejs16-image \
 	push-function-runtime-python39-image
 
-
```

Comment:
Why this Makefile consist only targets building two images? What with other runtimes?